css/style_card.css

```diff
@@ -0,0 +1,58 @@
+/* -------------------------------------------------------------------------- */
+/*                  Style of a card made the 12th of 08                       */
+/* -------------------------------------------------------------------------- */
+
+.Card-120824 {
+    display: flex;
+    align-items: center;
+    background-color: #ffffff;
+    border-radius: 8px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    padding: 24px;
+    width: 700px;
+    transition: transform 0.3s ease, box-shadow 0.3s ease;
+}
+
+.Card-120824:hover {
+    transform: scale(1.05);
+    box-shadow: 0 8px 16px rgba(0, 0, 0, 0.2);
+}
+
+.Picture-120824 {
+    width: 80px;
+    height: 80px;
+    border-radius: 50%;
+    margin-right: 20px;
+}
+
+.Container-120824 h2 {
+    margin: 0;
+    font-size: 1.5em;
+    color: #333;
+}
+
+.Container-120824 p {
+    margin: 8px 0 0;
+    color: #777;
+}
+
+/* -------------------------------------------------------------------------- */
+/*            To minimize the size of the cards when a phone is used.         */
+/* -------------------------------------------------------------------------- */
+
+@media (max-width: 500px) {
+    .card {
+        width: 400px;
+        padding: 15px;
+    }
+
+    .profile-photo {
+        width: 60px;
+        height: 60px;
+        margin-right: 15px;
+    }
+}
```

Comment:
The rules defined in this media query don't apply to any element on the page. To make them apply, you should change the class name in the selectors to have them match the class name of elements on the page (e.g. replace `.card` with `.Card-120824`).

Another thing you should change here is the media query condition. You're only applying this specific style when the device's width is less than or equal to 500 pixels, but with a screen width between 500 and 700 pixels, your `.Card-120824` element already extends beyond the viewport. I suggest you bring the `max-width` condition to 720 pixels (to keep some small margins on the sides) or create multiple media queries for more gradual scaling.